copilot-chat-instance.el

```diff
@@ -55,12 +55,8 @@ Use `copilot-chat-set-model' to interactively select a model."
  (spinner-timer nil :type timer)
  (spinner-index 0 :type int)
  (spinner-status nil :type (or null string))
- (curl-answer nil :type (or null string))
- (curl-file nil :type (or null file))
- (curl-current-data nil :type (or null string))
- (shell-maker-tmp-buf nil :type buffer)
- (shell-maker-answer-point nil :type point)
- (shell-cb-fn nil :type function))
+ (backend-private nil)
+ (frontend-private nil))
```

Comment:
Emacs Lisp Style to,

```suggestion
 (-backend nil)
 (-frontend nil))
```

And I should be write helper function and do not accsess private slot by other module.

Reply:
I changed the names but I don't know how to make them private.